kotlin/flatbuffers-kotlin/build.gradle.kts

```diff
@@ -1,29 +1,32 @@
+import org.gradle.internal.impldep.org.fusesource.jansi.AnsiRenderer.test
+import org.jetbrains.kotlin.gradle.plugin.mpp.apple.XCFramework
+import org.jetbrains.kotlin.cli.common.toBooleanLenient
+import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget
+import org.jetbrains.kotlin.gradle.plugin.mpp.NativeBuildType
+import org.jetbrains.kotlin.gradle.plugin.mpp.apple.XCFrameworkConfig
+
 plugins {
   kotlin("multiplatform")
 }
 
+
+val libName = "Flatbuffers"
 group = "com.google.flatbuffers.kotlin"
 version = "2.0.0-SNAPSHOT"
 
 kotlin {
   explicitApi()
   jvm()
-  js {
-    browser {
-     testTask {
-        useKarma {
-          useChromeHeadless()
-        }
-      }
-    }
-    binaries.executable()
-  }
   macosX64()
```

Comment:
Why was this removed? Does that mean no JS support from this Kotlin Multiplatform version?

Reply:
I was having some issues with it on mac, mostly npm. I removed temporarily to simplify the PR and will add it back later.